chore: add MEMBERSHIP_SURVEY.md file (#437)
nodejs/community-committee: governance/MEMBERSHIP_SURVEY.md

# Membership Survey

> Surveys for current and past members to collect data on member wants and needs. following #375 issue.

Reviewer: Can probably remove the final sentence:

```suggestion
Surveys for current and past members to collect data on member wants and needs.
```

## Suggested Questions

> @amiller-gh

Reviewer: I would remove the credits for the questions, since they're questions from the CommComm as a whole. Generally crediting is almost always better, but presenting a unified front is probably best in this specific case :)

> What is your current happiness (perceived value?) of Ind. Membership program?

Reviewer: Can change `Ind.` to Individual:

```suggestion
What is your current happiness (perceived value?) of Individual Membership program?
```

> Net promoter score – how likely would you recommend the program to a friend?

Reviewer: You can remove `Net promoter score` here since it's just describing what the question is commonly referred to:

```suggestion
How likely would you recommend the program to a friend?
```

> How did you hear about the IM program?

Reviewer: Can change `IM` to `Individual Membership`:

```suggestion
How did you hear about the Individual Membership program?
```

In your words, what value do you expect to receive from an open source individual membership?
(For people with lapsed membership) What made you not renew?

  

@joesepi

> Why did you become an Individual Member? (If lapsed, why?)

Reviewer: `(If lapsed, why?)` is a duplicate of the final question in @amiller-gh's section

> The goal of the Individual Membership program is to (insert clearly defined goal here); do you believe it is successful in that goal. If so/not, how/why?

Reviewer: We may want to iterate on `(insert clearly defined goal here)`. Perhaps @dshaw, @amiller-gh, and @joesepi have more context they'd like to share.

> How do you think the program could be improved?

Reviewer: I believe we've moved past this and it can be removed.
